src/devrules/cli_commands/prompters/gum_prompter.py

```
@@ -0,0 +1,193 @@
+
+        Returns:
+            User input string or None if cancelled
+        """
+        return gum.input_text(placeholder, header, default, char_limit)
+
+    def write(
+        self,
+        placeholder: str = "",
+        header: str = "",
+        char_limit: int = 0,
+        default: str = "",
```

Ruff (0.14.10)

Unused method argument: `default`

(ARG002)